mailpoet/tests/integration/Subscribers/ImportExport/Import/ImportTest.php

```diff
@@ -880,12 +879,7 @@ private function createSubscriber(
   }
 
   public function _after(): void {
-    $this->truncateEntity(SubscriberEntity::class);
-    $this->truncateEntity(SegmentEntity::class);
-    $this->truncateEntity(SubscriberSegmentEntity::class);
+    parent::_after();
     $this->truncateEntity(CustomFieldEntity::class);
```

Comment:
Did the custom field entity truncation stay here by mistake?

Reply:
Removed: https://github.com/mailpoet/mailpoet/pull/4818/commits/557577a432821dcc8238308bb4ff7b349b3907a4